pkgs/kde/gear/akonadi/default.nix

```diff
@@ -22,4 +22,10 @@ mkKdeDerivation {
 
   extraNativeBuildInputs = [qttools shared-mime-info];
   extraBuildInputs = [kaccounts-integration accounts-qt xz mariadb];
+
+  # Hardcoded as a QString, which is UTF-16 so Nix can't pick it up automatically
+  postFixup = ''
+    mkdir -p $out/nix-support
+    echo "${mariadb}" > $out/nix-support/depends
```

Comment:
Do we need out or lib here? I am not sure

Reply:
Out, it wants the binaries